src/components/Main/Main.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './Main.css';
+import backgroundImage from '../../background.png';
+import Animal from '../Animal/Animal';
 
-export default function Main() {
-  return <main></main>;
+
+
+// whats up
+
+export default function Main({ animals }) {
+  return ( 
+    <main className= 'background' style={{ backgroundImage: `url(${ backgroundImage })` }}>;
+
+      {animals.map((animal, i) => <Animal key={animal.name + i} {...animal} />)}
```

Comment:
This mapping, and rendering of the Animal component needs to be abstracted into a separate AnimalList component that gets called here, and passed the animals array as a prop